### PR TITLE
Add task for terra backend env exploration

### PR DESCRIPTION
**Pull request by tomkinsc**

### CODE REVIEW

File: pipes/WDL/tasks/tasks_terra.wdl [hunk 1]
```diff
@@ -30,6 +30,42 @@ task gcs_copy {
...
+  }
+  meta {
+    description: "task for inspection of backend env, and optionally running a command"
+  }
+  parameter_meta {
+    additional_command_to_run: {
+      description: "Command that can optionally be included and executed as part of this task"
+    }
+  }
+  command {
```
Comment (dpark01) [inline pipes/WDL/tasks/tasks_terra.wdl:45]:
```suggestion
  command <<<
```

Comment (dpark01) [reply pipes/WDL/tasks/tasks_terra.wdl:45]: probably need to switch syntax?

**Review by dpark01 — COMMENTED** [review]
Just to clarify, this isn't actually a PR against `master`, you're just playing in a branch to grab some reconnaissance / telemetry from Terra, but can the PR be closed?

File: pipes/WDL/tasks/tasks_terra.wdl (hunk 1 above)
Comment (tomkinsc) [reply pipes/WDL/tasks/tasks_terra.wdl:45]: Are we standardizing toward `<<<` to [discourage](https://github.com/openwdl/wdl/blob/main/versions/1.0/SPEC.md#expression-placeholders) WDL interpolation of`${}`?

**Comment by tomkinsc** [comment]
Yeah, this is all for inspection and not intended to be merged (unless we want to preserve some ability to inspect the environment?)

File: pipes/WDL/tasks/tasks_terra.wdl (hunk 1 above)
Comment (dpark01) [reply pipes/WDL/tasks/tasks_terra.wdl:45]: Yes there's definitely a number of bashisms that simply won't work with the older WDL syntax

**Comment by dpark01** [comment]
Ok, recommend closing this PR, as it's not necessary for the build, CI testing, or Terra testing

**Pull request closed without merging**